src/Raven.Client/Documents/Subscriptions/SubscriptionState.cs

```diff
@@ -17,17 +15,20 @@ public class SubscriptionState : IDatabaseTask, IDatabaseTaskStatus
     {
         public string Query { get; set; }
         public string ChangeVectorForNextBatchStartingPoint { get; set; }
-        public Dictionary<string, string> ChangeVectorForNextBatchStartingPointPerShard { get; set; }
         public long SubscriptionId { get; set; }
         public string SubscriptionName { get; set; }
         public string MentorNode { get; set; }
         public bool PinToMentorNode { get; set; }
-        public string NodeTag { get; set; }
+
         public DateTime? LastBatchAckTime { get; set; }  // Last time server made some progress with the subscriptions docs  
         public DateTime? LastClientConnectionTime { get; set; } // Last time any client has connected to server (connection dead or alive)
-        
         public bool Disabled { get; set; }
 
+        // in sharding context - orchestrator node tag
+        public string NodeTag { get; set; }
+        
+        public SubscriptionShardingState SubscriptionShardingState { get; set; }
```

Comment:
rename property to SubscriptionShardingState  => ShardingState

Reply:
done